src/pages/index.js

```diff
@@ -0,0 +1,305 @@
+import "./index.css";
+import Api from "../utils/Api.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "1810d547-91a3-4cf2-9027-84faa0c5dd2e",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, users]) => {
+    cards.forEach((card) => {
+      const cardEl = getCardElement(card);
```

Comment:
` const cardEl = getCardElement(card);` is not needed here because everything is done inside ` renderCard(card);`